lib/elixir/lib/enum.ex

```diff
@@ -2143,28 +2143,66 @@ defmodule Enum do
 
   @doc """
   Returns a tuple with the minimal and the maximal elements in the
-  enumerable according to Erlang's term ordering.
+  enumerable.
 
   If multiple elements are considered maximal or minimal, the first one
   that was found is returned.
```

Comment:
```suggestion
  By default, the comparison is done with the `<=` sorter function.
  If multiple elements are considered maximal or minimal, the first
  one that was found is returned.
```

Reply:
I have updated the text here. The code won't work if it the sorting function returns equal. Can you please apply it accordingly to `min_max_by`? Then we are good to go.